nevergrad/optimization/test_optimizerlib.py

```diff
@@ -129,6 +129,7 @@ def test_optimizers(name: str) -> None:
     if isinstance(optimizer_cls, base.ConfiguredOptimizer):
         assert any(hasattr(mod, name) for mod in (optlib, xpvariants))  # make sure registration matches name in optlib/xpvariants
     verify = not optimizer_cls.one_shot and name not in SLOW and not any(x in name for x in ["BO", "Discrete"])
+    print("Alors : {} et {}".format(name, verify))
```

Comment:
please remove this

Reply:
Oups sorry ; I missed that during my cleaning!